jetty9/src/main/docker/jetty_start.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+DIR=$(dirname $0)
+JETTY_START_COMMAND="/scripts/jetty/jetty-start-command"
+
+if [ ! -f $JETTY_START_COMMAND ]; then
+  source $DIR/gen_start_command.sh $JETTY_START_COMMAND
```

Comment:
oh absolutely, i just saw that comment about doing something for people sensitive to container start time and wanted to chime in 👍